### DISCUSSION TITLE
Welcome to GeoVista Discussions!

### DISCUSSION THREAD

**Discussion by bjlittle** [bjlittle/geovista#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tkoyama010** [comment]
Congratulation! [The number of stars of GeoVista](https://github.com/bjlittle/geovista/stargazers) will reach 100. This project is awesome.

**Reply by bjlittle** [reply]
We're sooo excited! It's incredible 🤩

**Comment by bjlittle** [comment]
![image](https://github.com/bjlittle/geovista/assets/2051656/3ca5f653-34c7-4af3-85e0-4d03b95423c1)

Cheers to all our first 100 amazing stargazers 🤩🥳🚀 

![oktoberfest](https://github.com/bjlittle/geovista/assets/2051656/7f1e9598-a1ba-4d04-80d0-4e22f1c924e8)

**Reply by tkoyama010** [reply]
@bjlittle  I made a [PR](https://github.com/tkoyama010/awesome-python-dataviz/pull/20) of adding GeoVista in my [awesome-python-dataviz](https://github.com/tkoyama010/awesome-python-dataviz) repository. I guess this project is worth to add awesome list.

**Comment by berndbecker** [comment]
HI! 
My name is Bernd Becker and I came up with the geo point cloud in the ORCA12 example here:
https://geovista.readthedocs.io/en/latest/generated/gallery/point_cloud/from_points__orca_cloud_eqc.html#sphx-glr-generated-gallery-point-cloud-from-points-orca-cloud-eqc-py

I am saving these point clouds as is in vtk files and wonder if it was not possible to provide a re-transfer bridging function to 
revert the points back into human readable lon, lat and depth?

We have the zscale and the zlevel . The geovista.common.to_cartesian function is used by the from_points method to create the cartesian xyz points, so given what we  know,  undoing the z values to yield the original depths should also be straightforward.

**Reply by bjlittle** [reply]
Hey @berndbecker,

Great to hear from you :smile: 

This sounds like a great enhancement, so I've created this issue https://github.com/bjlittle/geovista/issues/1029 to capture your use case and move things forward :+1:
